### ISSUE TITLE
Handle different config references by default

### ISSUE THREAD

**Issue by dekelpilli** [AVSystem/intellij-hocon#26]
This is for showing uses across a project. Example syntaxes I'm familiar with:

* {{}} and ${} for Camel
* ${} and #{} for Spring

For example:
[This works](https://i.imgur.com/vSmyFNr.png)
[This doesn't work](https://i.imgur.com/LdTNDGP.png)

**Comment by ghik** [comment]
I agree that this would be useful, but I think this is out of scope of the HOCON plugin. Since we are dealing with Camel/Spring-specifix syntax, this should be provided by plugins supporting these frameworks (or perhaps specialized separate plugins which integrate Camel/Spring with HOCON).

**Comment by dekelpilli** [comment]
That's fair. I didn't mean to suggest that this plugin would try to detect the presence of these frameworks and their valid use cases, but maybe just assume that config surrounded by any `{}` anywhere match. It could give a few false positives, but that's probably better than mostly false negatives.
